docs/design/resources.md

```diff
@@ -63,21 +63,21 @@ following:
    successfully connected update the `status` field with information
    from the cluster.
 3. If we have communication with the cluster and the cluster is
-   quorate, check that the number of peers the cluster expects is the
-   same as the cluster resource's specified `size`. If the cluster has
-   too few peers, add a new one using the API. If too many, remove
-   exactly using the API.
-3. List the `EtcdPeer` resources that are our children. If we have
+   quorate, check that the number of members the cluster expects is
+   the same as the cluster resource's specified `size`. If the cluster
+   has too few members, add a new one using the API. If too many,
+   remove exactly one using the API.
+4. List the `EtcdPeer` resources that are our children. If we have
    communication with the cluster reconcile the peer resources with
-   the peer list in etcd. Peers that the exist in Kuberentes that the
-   cluster does not know about should be deleted, peers that exist in
-   the cluster and are not in Kubernetes should be created.
+   the member list in etcd. Peers that the exist in Kuberentes that
+   the cluster does not know about should be deleted, members that
+   exist in the cluster and are not in Kubernetes should be created.
```

Comment:
`and do not have corresponding EtcdPeer resources` perhaps?

Reply:
This whole section was a little confusing. I updated it to read:

```
4. List the `EtcdPeer` resources that are our children. If we have
   communication with the cluster, reconcile the peer resources with
   the member list in etcd:

   * If a member exists in the etcd member list, but does not have a
     corresponding `EtcdPeer`, then create the peer resource.
   * If a `EtcdPeer` resource exists that does not have a
     corresponding member in the etcd memeber list, then remove the
     peer resource.
```

Reply:
Updated in bd764b7